README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
@ljharb I'm mostly concerned with clarifying the install which seems to be accounted for here: https://github.com/airbnb/react-dates/pull/168 so does not like I'll need to submit a PR anymore ... the rest of this stuff should be clarified by examples and is pretty standard